output/schema/schema.json

```diff
@@ -17080,6 +17087,256 @@
         "namespace": "_global.info"
       }
     },
+    {
+      "attachedBehaviors": [
+        "CommonQueryParameters"
+      ],
+      "body": {
+        "kind": "properties",
+        "properties": [
+          {
+            "description": "Array of wildcard (*) patterns. The request returns values for field names\nmatching these patterns in the hits.fields property of the response.",
```

Comment:
Tiny comment, presenting `fields` as primarily an "array of wildcard patterns" is a bit surprising, since by far the most common use case is to provide field names with no wildcards. It's a fairly minor detail that it also accepts field wildcard patterns.